randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -2116,11 +2130,9 @@ Extra - asset_id: collision_camera_025_B
   * Extra - right_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:doorshieldmissile_001
   * Extra - right_shield_def: actordef:actors/props/doorshieldmissile/charclasses/doorshieldmissile.bmsad
   > Door from Chozo Warrior Arena
-      Morph Ball and Before Ghavoran - Ferenia Transport Grapple Box
+      Before Ghavoran - Ferenia Transport Grapple Box and Can Slide
```

Comment:
Same here, this distance is too far to slide alone - keep this as morph ball

Reply:
And as before, you'll just keep sliding.